modules/src/main/modules/search/elasticsearch/module.xml

```diff
@@ -18,5 +18,9 @@
         <module name="org.hibernate.commons-annotations"/>
         <module name="org.jboss.logging"/>
         <module name="org.apache.httpcomponents"/>
+
+        <!-- AWS authentication support -->
+        <!-- Note: we must export the services (hence the classes), because services are loaded using the -engine classloader -->
+        <module name="org.hibernate.search.elasticsearch-aws" export="true" services="export" optional="true" slot="${hibernate.search.version}" />
```

Comment:
Unfortunately I am... If I don't export classes too, the service file (`META-INF/services/org.hibernate.search.elasticsearch.client.spi.ElasticsearchHttpClientConfigurer`) will be found, but the implementation mentioned in this file will not, and the deployment will fail: https://gist.github.com/yrodiere/420424979363cc07f3adc82cd8bf55b7
I find it weird, but I suppose it makes sense in some way.